CHANGELOG.md

```diff
@@ -24,7 +24,90 @@ Current
 - [`ResponseValidationException` uses `Response.StatusType` rather than `Response.Status`](https://github.com/yahoo/fili/pull/96)
     * `Response.StatusType` is the interface that `Response.Status` implements.
     * This will have no impact on current code in Fili that uses `ResponseValidationException`, and it allows customers to inject http
-        codes not included in `Response.Status`. 
+        codes not included in `Response.Status`.
+         
+- [Removed "provided" modifier for SLF4J and Logback dependencies in the Wikipedia example](https://github.com/yahoo/fili/pull/102)
```

Comment:
Do we put new stuff at the bottom or the top of the CHANGELOG? Because I've been putting them at the top...